src/linked-list.js

```diff
@@ -4,6 +4,39 @@ class LinkedList {
     this.tail = null;
     // Do not modify anything inside of the constructor
   }
+
+  addToTail(data) {
+    const newNode = {
+      value: data,
+      next: null,
+    };
+    if (this.head === null) {
+      this.head = newNode;
+      this.tail = newNode;
+    } else if (this.head.next === null) {
```

Comment:
You could eliminate this else if statement, as it's doing the same thing as your else statement below.